File: Aulas/U01_Nivelamento/U00o/01 - Matriz em Java/Ex01.java
/**
 * Matriz em Java
 * @author Camila Moreira Lopes
 */

public class Ex01{
    public static void main (String[] args){
        int x = MyIO.readInt();
        int[][] matriz = new int[x][x];
        int[] soma = new int[x];

        for(int i = 0; i < x; i++){
            for(int j = 0; j < x; j++){
                matriz[i][j] = MyIO.readInt();
            }
        }

        for(int i = 0; i < x; i++){
            soma[i] = 0;
            for(int j = 0; j < x; j++){
                soma[i] += matriz[j][i];
            }
        }
        for(int i = 0; i < x; i++){
            System.out.print(soma[i] + " ");
        }
    }
}

File: Aulas/U01_Nivelamento/U00m/ex01.c
/**
 * @author Camila Moreira Lopes
 * Faça um programa que leia n números inteiros, armazene-os em um arquivo, leia-os do arquivo e mostre-os na tela
 */

#include <stdio.h>
#include <stdlib.h>
#include <string.h>


void main(int argc, char** argv){
    int n = scanf("%i", &n);
    int vetor[n];
    for(int i = 0; i < n; i++){
        scanf("%i", &vetor[i]);
    }
    
    FILE *p = fopen("ex01.txt", "wd");
    for(int i = 0; i < n; i++){
        fwrite(&vetor[i], sizeof(int),1,p);
    }
    fclose(p);

}